Makefile.standard_app

```diff
@@ -79,6 +79,38 @@ ifneq ($(DISABLE_DEFAULT_IO_SEPROXY_BUFFER_SIZE), 1)
     endif
 endif
 
+#####################################################################
+#                              NBGL                                 #
+#####################################################################
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_NANOS TARGET_NANOX TARGET_NANOS2))
+ifeq ($(ENABLE_NBGL_FOR_NANO_DEVICES), 1)
+USE_NBGL = 1
+else
+USE_NBGL = 0
+endif
+else
+USE_NBGL = 1
+endif
+
+ifeq ($(ENABLE_NBGL_QRCODE), 1)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_STAX TARGET_FLEX))
+    DEFINES += NBGL_QRCODE
+    SDK_SOURCE_PATH += qrcode
+endif
+endif
+
+ifeq ($(ENABLE_NBGL_KEYBOARD), 1)
+ifeq ($(TARGET_NAME),$(filter $(TARGET_NAME),TARGET_STAX TARGET_FLEX))
```

Comment:
NBGL_KEYBOARD & NBGL_KEYPAD exist also on Nanos

Reply:
Yeap but as of now the corresponding useCase functions are not implemented.
So we can probably keep this like it for now.

Reply:
Fixed